includes/adminkey.pub

```diff
@@ -0,0 +1 @@
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQDeEZqjLjbZmqFrhYL1inJM/U92m1/rZ+Lq/ZJvsEITk0I/7GBJqZ5Ze4Oyh/XKtbCC3l9QTwfhOyd/jlcjIaKaRPznY0Prml11lMY6OJHLPaBvKISaRoZb/J9xk0fwRAuXoVQNKMCE5DXCIBYNWEILex6Z2MzESRwbsEciDxdIuBSsJA9IPfCrFZhrgm/oBCtvEKflQCwxMcBI3kdfPUInFpqpafvvNJl0uIexsRMrKEyUMJ7+WDVXc/UQm4P/dodiLPXZkppwrPg5OzqIZWW5TZglCtlo5hMOMD4fTyKplTwWXliKtrGrb54lELCHJpSGDhx/y9aeX+mLXszXeC/QFWPg3DWaFpbaix6NGTR6cDDKpoUKau5hiu03AYz9EJgKDh3FZ/VgSZ5r4FLrU5zErdbyuipc1hcPzc4iEz5oKPxrPK0wvW7zXCibFsYruTxQkFWvF16nLDhbCQ97FjGFVRMP4e5v3V3W1hm4SOqXVfbDpukr6FXYWoeAtJZZpYNKlIkeeOdqFokisZLCpJPjo949n6pgXbyuaDtaArvgcvoIFtKKwfDOrY078vyyX4WsE4kDM3JSGHdkcNO52Ofe84rvFLRuIoRlZEuIfvhTRc6EMgQ89MDZJI4cMb6AzzS/okvKTSZoyualIZW+2fET1fRwSiH7XwLPgg3LpZkybQ== Markus Storm
```

Comment:
This should not be a change included in the master branch, right? Unless you want your key publicly available.

Reply:
right, this was for testing only, will remove